backend/app/src/main/java/com/ugent/pidgeon/util/DockerClientInstance.java

```diff
@@ -21,8 +21,8 @@ private DockerClientInstance() {
   public static synchronized DockerClient getInstance() {
 
     if (dockerClient == null) {
-      DockerClientConfig config = DefaultDockerClientConfig.createDefaultConfigBuilder()
-          .withDockerHost("tcp://10.5.0.4:2375").build();
+
+      DockerClientConfig config = DefaultDockerClientConfig.createDefaultConfigBuilder().build();
```

Comment:
kweet niet of dit de bedoeling was, was dit niet net aangepast om dind te werken? Of werkt het dan niet lokaal?

Reply:
Dit was wel de bedoeling, deze code van arthur werkte niet meer en hij moest hier nog een mail voor sturen. Op deze manier werkt het lokaal nog.